src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -329,11 +328,10 @@ export class HaConfigDevicePage extends LitElement {
     const entitiesByCategory = this._entitiesByCategory(entities);
     const batteryEntity = this._batteryEntity(entities);
     const batteryChargingEntity = this._batteryChargingEntity(entities);
-    const batteryState = batteryEntity
+    const battery = batteryEntity
```

Comment:
Why do we rename the const here?

Reply:
It was renamed to be consistent with the other 2 pages. (The code is being used on 3 pages and is mostly the same, I've looked into putting it in it's own component. )